docsource/use_cases/module_renaming.rst

```diff
@@ -0,0 +1,169 @@
+""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. csv-table::
+   :header: "id", "name", "state", "latest_version"
+
+   "200", "crm_iap_lead_enrich", "to upgrade", "14.0.1.1"
+   "405", "crm_iap_enrich", "uninstalled", ""
+
+
+**Problem**:
+
+- The v14 module remains in a ``to upgrade`` state.
+- The v15 module has not been installed.
+
+As a result, the models of the module have not been loaded, the related features
+are not available, and the database is in a unstable state.
+
+V12 table ``ir_module_module`` (With openupgrade)
+"""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. csv-table::
+   :header: "id", "create_date", "write_date"
+
+   "200", "crm_iap_enrich", "installed", "15.0.1.1"
+
+Contribution to OpenUpgrade
+---------------------------
+
+apriori.py file
+"""""""""""""""
+
+The list of the renamed module is present in a ``apriori.py`` file in each version
+of openupgrade.
+
+* Until version 13, the file is present in this path:
+  ``odoo/addons/openupgrade_records/lib/apriori.py``.
+
+* from version 14.0 onwards, the file is present in this path:
+  ``openupgrade_scripts/apriori.py``.
+
+Add a key in the following dict:
+
+.. code-block:: python
+
+    renamed_modules = {
+        "crm_iap_lead_enrich": "crm_iap_enrich",
+    }
+
+See `Full v15 apriori file <https://github.com/OCA/OpenUpgrade/blob/97491cb7d9a8ed494a49cf1db9b7fc8852aac254/openupgrade_scripts/apriori.py#L9-L27>`_.
+
+pre-migration.py file
+"""""""""""""""""""""
+
+in the migration folder of the ``base`` module, add:
+
+.. code-block:: python
+
+    from odoo.addons.openupgrade_scripts.apriori import renamed_modules
+
+    @openupgrade.migrate(use_env=False)
+    def migrate(cr, version):
+        openupgrade.update_module_names(cr, renamed_modules.items())
+
+See `Full v15 base pre migration file <https://github.com/OCA/OpenUpgrade/blob/97491cb7d9a8ed494a49cf1db9b7fc8852aac254/openupgrade_scripts/scripts/base/15.0.1.3/pre-migration.py#L73>`_.
+
+Note: The addition is done at the initialization of the OpenUpgrade Project,
+for each new version. You will not have to do this,
+if you are working on a version of OpenUpgrade under development.
+
+Coverage File
+"""""""""""""
+
+A coverage file is present in each version:
+
+* Until version 13, the file is present in this path:
+  ``odoo/openupgrade/doc/source/modulesXXX-YYY.py``.
+
+* from version 14.0 onwards, the file is present in this path:
+  ``docsource/modulesXXX-YYY.py``.
+
+Edit this file, and add the following text:
+
+.. code-block:: rst
+
+    +-----------------------------------+-------------------------------------------------+
+    | |new| crm_iap_enrich              | Renamed from crm_iap_lead_enrich                |
+    +-----------------------------------+-------------------------------------------------+
+    | |del| crm_iap_lead_enrich         | Renamed to crm_iap_enrich                       |
+    +-----------------------------------+-------------------------------------------------+
```

Comment:
There is 3 columns in the file `modulesXXX-YYY.rst`. So I think it's better to propose an example with 3 columns, even if the last one is left blank. Or you can name the column so that there is no ambiguity in which column to put the rename information.

Reply:
header and third column added. Thanks for your remark.